st2debug/st2debug/cmd/submit_debug_info.py

```diff
@@ -38,6 +38,7 @@
 import platform
 import tempfile
 import httplib
+import ConfigParser
```

Comment:
Does this file use tabs instead of spaces? If so might not hurt to convert it to spaces per their coding standards:
https://docs.stackstorm.com/development/index.html